docs/book/src/plugins/declarative-v1.md

```diff
@@ -1,31 +1,30 @@
 # Declarative Plugin
 
```

Comment:
@Kavinjsir,

I just checked we also need to update the docs for grafana right: https://github.com/kubernetes-sigs/kubebuilder/blob/31f6eb9be3ab3c62573ce4ea384603907b286a9f/docs/book/src/plugins/grafana-v1-alpha.md

Reply:
That's a nice catch 👍🏼 

Just fixed that. Thx for pointing out!